dev/source/docs/sitl-with-webots.rst

```diff
@@ -33,8 +33,8 @@ Running the Examples
 .. warning::
 
    It is important to run Webots before ArduPilot's SITL , otherwise the socket connection may not establish.
-
-
+   
+   As of writing, these instructions work with Webots 2021b. Newer versions may or may not work correctly.
```

Comment:
Newer versions will not work correctly.
I am in the process of preparing one for R2023a.

Reply:
@HefnySco so should we merge this as is or wait?

Reply:
I suggest we can merge this, and we will keep this next to the newer version to keep compatibility with older versions in case someone  is using this old version with his/her world files.